### ISSUE TITLE
Enable addition of accounts for Ledger users

### ISSUE THREAD

**Issue by markmhendrickson** [blockstack/blockstack-app#2502]
The first version of Ledger support enables only a pre-determined number of accounts (5). We should enhance this to allow the user to add any greater number of accounts they wish.

**Comment by pete-watters** [comment]
Some power users have been having issues with this so we agreed to add a simple way for them to load more accounts on Ledger.

**Comment by markmhendrickson** [comment]
@kyranjamie might supporting this be as simple as adding a text input to the Ledger connection sheet (modal) that displays the number of accounts to load (for approval on the device, defaulting to 10), giving the user the opportunity to modify the number if they want more or fewer accounts before Leather requests from the device just how many to approve for connection?

Note that we support 10 these days for all users instead of 5 as originally supported. We occasionally have users asking for more than 10. I assume the main UX issue with simply increasing that universal number further (e.g. to 20) is that all Ledger users would have to wait longer upon connection, even if they don't want more accounts.

**Comment by kyranjamie** [comment]
Yes, that would be a fairly simple change. More so than adding an _import more accounts_ later flow.
